Sources/LeafKit/Exports.swift

```diff
@@ -33,6 +39,22 @@ extension Character {
     static let vertical = "|".first!
     static let underscore = "_".first!
     
+    var isHexidecimal: Bool {
+        return (.zero ... .nine).contains(self)
+            || (.A ... .F).contains(self.uppercased().first!)
+            || self == .hexNotation
+    }
+    
+    var isOctal: Bool {
```

Comment:
Admittedly I only did it since it didn't require any extra work after bin/hex, but I can see having shorthand for permission checks being useful anyway - templates for directory views would certainly want to be able to branch say, disabling URL links on non-readable files